Cabal/src/Distribution/Simple/PreProcess.hs

```diff
@@ -24,7 +24,7 @@ module Distribution.Simple.PreProcess (preprocessComponent, preprocessExtras,
                                 knownSuffixHandlers, ppSuffixes,
                                 PPSuffixHandler, PreProcessor(..),
                                 mkSimplePreProcessor, runSimplePreProcessor,
-                                ppCpp, ppCpp', ppGreenCard, ppC2hs, ppHsc2hs,
+                                ppCpp, ppCpp', ppGhcCpp, ppGreenCard, ppC2hs, ppHsc2hs,
```

Comment:
What's this export for? Is the function now unused and could be deleted instead?

Reply:
I figured users might want it - seemed safest to allow it in custom `Setup.hs` if they really want.

Reply:
Hmm I'd tend to not extend the API unless there's a clear need (and we add some test coverage). But it's not for me to decide.